src/components/Contractor/Submit/Submit.tsx

```diff
@@ -76,7 +76,7 @@ export const Root = ({ contractorId, selfOnboarding }: ContractorSubmitProps) =>
     return <SubmitDone onDone={handleSubmitDone} />
   }
   if (
-    onboardingStatus === ContractorOnboardingStatus.ADMIN_ONBOARDING_INCOMPLETE &&
+    onboardingStatus === ContractorOnboardingStatus.SELF_ONBOARDING_NOT_INVITED &&
```

Comment:
for my understanding, why are we changing the enum check here? and is there ever a case where multiple enums would qualify as truthy for this check?

Reply:
I think this was messed up previously due to the contractor update endpoint being called incorrectly in the file new hire report. This was a needed change after i corrected that

For the other statuses we wouldn't want to show the invite screen... unless they were changing the recipient email, but that's a case we would probably need some UX around anyway

This is probably still something that will need work on refining moving forward separately as contractor onboarding gets traction with partners, but this at least makes the happy path functional for now.